doc/quick-install.md

```diff
@@ -22,50 +20,43 @@ Then add the same FQDN in the `/etc/hosts` file, associating it with the loopbac
 
 Finally on the same machine test that you can ping the FQDN with: `ping "$(hostname)"`-
 
-### Add the repository
+### Add the Jitsi package repository
 ```sh
 echo 'deb https://download.jitsi.org stable/' >> /etc/apt/sources.list.d/jitsi-stable.list
 wget -qO -  https://download.jitsi.org/jitsi-key.gpg.key | apt-key add -
 ```
 
-### Update the package lists
+### Generate a Let's Encrypt certificate (optional, recommended)
 
```

Comment:
Why did you move this up in the list, it should be after installing jitsi-meet?

Reply:
The installer says to the user to run let's encrypt after installing jitsi-meet.

Reply:
My idea here was that we could have users prepare their Let's Encrypt certificate in advance, and then they'd be able to select it directly during the installer, without having to 'go back around' and waste a few cycles on self-signed certificate generation.

(Ok, that's slightly tongue-in-cheek about the wasted CPU cycles; but if we can save a few steps then I figure we should)

The installer text could remain as-is for anyone following separate install guidelines, accidentally misses this step, or changes their mind and decides to use Let's Encrypt after starting the installer.

Reply:
That will not work. That script asks just for email and grabs the rest of the information from the installed jitsi-meet. That was the idea of that automation. Otherwise, we can just remove the script and just add a note, "hey do the let's encrypt thing" and tie a link to their howto.
Maybe when you are there add a note here that mobile will not work without this step! I think that is also important.

Reply:
Understood; I'll relocate the Let's Encrypt instructions to their original position, and then add that notice regarding mobile clients.